pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "radiacode"
-version = "0.3.3"
-description = "Library for RadiaCode-101"
+version = "0.3.4"
```

Comment:
The current release pipeline includes setting an annotated tag and pushing to GitHub to build a pip package with GitHub Actions.
I will publish the release later, so please revert the version bump (but changes in the description are okay).